lib/cmd/common/initlocal.js

```diff
@@ -125,64 +128,82 @@ function modifyTagsToLocal(appContainer, tagsToModify, localDirectory, cb) {
 function constructContainerFromIndex(indexFile, cb) {
   var outputFile;
   var appContainer = cheerio.load(indexFile);
-  setBodyToPlaceholder(appContainer, function (err, appContainer) {
-    if (err) return cb(err);
+  console.log(appContainer.html());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)